src/content/getting_started/architecture/_index.en.md

```diff
@@ -19,3 +20,25 @@ both on-premises and on public clouds:
 Submariner has optional components that provide additional functionality:
 
 * [Globalnet Controller](./globalnet/): handles interconnection of clusters with overlapping CIDRs.
+
+The diagram below illustrates the basic architecture of Submariner:
+
+![Submariner Architecture](/images/submariner/architecture.jpg)
+
+### Terminology and Concepts
+
+* Clusterset - a group of two or more clusters with a high degree of mutual trust that share Services among themselves.
+Within a clusterset, all namespaces with a given name are considered to be the same namespace.
```

Comment:
```suggestion
* `ClusterSet` - a group of two or more clusters with a high degree of mutual trust that share Services among themselves.
Within a `ClusterSet`, all namespaces with a given name are considered to be the same namespace.
```

or

```suggestion
* `ClusterSet` - a group of two or more clusters with a high degree of mutual trust that share Services among themselves.
Within a cluster set, all namespaces with a given name are considered to be the same namespace.
```

https://submariner.io/for_developers/dev_guide/website/style_guide/#submarinerio-word-list

There are a few other places in this PR that should also use `ClusterSet` or "cluster set" imho.

Reply:
Thanks @dfarrell07. I remember that we discussed this in the past, but I was looking at the upstream spec again (https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api) and it seems to be using "clusterset" and not "cluster set".

Reply:
Yeah, I saw that too but I think they are just wrong there. It's a draft with a fair number of small errors. Once it's official, I suspect they will follow the main K8s guidance for API objects:

https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects